config-linux.md

```diff
@@ -726,6 +726,7 @@ The following parameters can be specified to set up seccomp:
     * `SCMP_ARCH_PARISC`
     * `SCMP_ARCH_PARISC64`
     * `SCMP_ARCH_RISCV64`
+    * `SCMP_ARCH_LOONGARCH64`
```

Comment:
> L708: A valid list of constants as of libseccomp v2.5.0 is shown below.

This has to be updated to v2.6.0 (and we have to wait for v2.6.0 to be released)